modin/experimental/engines/omnisci_on_native/test/test_dataframe.py

```diff
@@ -654,7 +655,17 @@ def concat(df1, df2, lib, **kwargs):
             df4 = df2.copy()
             return lib.concat([df1, df2, df3, df4])
 
-        run_and_compare(concat, data=self.data, data2=self.data2)
+        def sort_comparator(df1, df2):
+            """Sort and verify equality of the passed frames."""
+            # We sort values because order of rows in the 'union all' result is inconsistent in OmniSci
+            df1, df2 = map(
+                lambda df: try_cast_to_pandas(df).sort_values(df.columns[0]), (df1, df2)
+            )
```

Comment:
are you telling me this is easier to read than a pure list comprehension?..
```suggestion
            df1, df2 = [try_cast_to_pandas(df).sort_values(df.columns[0]) for df in (df1, df2)]
```

Reply:
changed to a list comprehension